ee/localserver/request-id.go

```diff
@@ -20,8 +23,9 @@ type identifiers struct {
 type requestIdsResponse struct {
 	RequestId string
 	identifiers
-	Nonce     string
-	Timestamp time.Time
+	Nonce        string
+	Timestamp    time.Time
+	ConsoleUsers []*user.User
```

Comment:
I wonder if this is ever going to be more than one. We'll see!

Reply:
This is a little tricky

it's possibly there will be zero `ConsoleUsers`

There appear to [be some windows service](https://learn.microsoft.com/en-us/windows-server/remote/remote-desktop-services/welcome-to-rds) that allow "Session-based virtualization" for multiple users. I have no idea how this works under the hood. Virtualization implies it's a VM on the machine, then what user owns that VM? There would a lot of research here. If they login from the VM then I guess it doesn't matter? It's also for windows server, which feels like an edge case.

I could be persuaded to drop it to just a singular `ConsoleUser`, but this it just felt safer to have it plural so that downstream consumers since I don't fell 100% confident there will always be just 1.

Reply:
That's reasonable. Let's keep it.